src/components/login/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
```

Comment:
We're requiring jQuery 2+ which only supports ie9+ so I think we can skip the extra checks